homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -70,6 +70,7 @@ class PlugwiseBinarySensorEntityDescription(BinarySensorEntityDescription):
         icon="mdi:mailbox-up-outline",
         icon_off="mdi:mailbox-outline",
         entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
```

Comment:
Why is this one disabled?

Reply:
On some devices there's almost never a notification. Also, several users have indicated that they were not interested in binary-sensor.

Reply:
It is marked as diagnostic already, which is for that purpose. It should not be disabled.

Reply:
Then I prefer to remove the entity-category.

Reply:
No, the entity category is correct, as this is not a primary entity.
Looking at the general architecture, this change is incorrect and should be reverted.

Reply:
OK, will do.